CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # webkms-client ChangeLog
 
+## 4.0.0 -
+
+### Changed
+- **BREAKING**: Update to use `crypto-ld v4` and to use
+  `Ed25519VerificationKey2020` key type for invocation signing.
```

Comment:
I gather that what's happening is that the default signature is changing from `Ed25519VerificationKey2018` to `Ed25519VerificationKey2020`.  Can more explicit language be added that this is the reason for the breaking change.

Reply:
No prob, will edit to emphasize.

Reply:
Reworded, thanks.